### PR TITLE
fix: mobile dialog design

### PR DESCRIPTION
**Pull request by edmonday**
# Description

copilot:summary

- [Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/36562315/todos/7145672740)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Walkthrough

copilot:walkthrough

### CODE REVIEW

File: apps/journeys-admin/src/components/TemplateView/AccountCheckDialog/AccountCheckDialog.tsx [hunk 1]
```diff
@@ -22,40 +24,57 @@ export function AccountCheckDialog({
...
-          fullWidth
-          onClick={handleSignIn}
-        >
-          {t('Create a new account')}
-        </Button>
+      <Stack
+        gap={8}
+        sx={{
+          alignItems: 'center'
+        }}
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/TemplateView/AccountCheckDialog/AccountCheckDialog.tsx:46]: Stack accepts the prop `alignItems="center"`

Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/TemplateView/AccountCheckDialog/AccountCheckDialog.tsx:46]: Can do without the sx, just looks cleaner that way

File: apps/journeys-admin/src/components/TemplateView/AccountCheckDialog/AccountCheckDialog.tsx [hunk 2]
```diff
@@ -22,40 +24,57 @@ export function AccountCheckDialog({
...
+          </Typography>
+        </Stack>
+        <Stack gap={5} sx={{ width: '100%' }}>
+          <Button
+            startIcon={<Key2 />}
+            variant="outlined"
+            size="large"
+            color="secondary"
+            onClick={handleSignIn}
+            sx={{ border: 1.5, borderRadius: 2, height: 50 }}
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/TemplateView/AccountCheckDialog/AccountCheckDialog.tsx:63]: Love that you're following the design, though if the buttons they introduced in the design doesn't exist within our set of buttons. Then don't follow design, meaning don't adjust the button through css. This goes back to the idea of following our design system. We will have to create this button as an option within the design system to be reused. For now just use one of our existing ones. Same goes for the button below please

Comment (edmonday) [reply apps/journeys-admin/src/components/TemplateView/AccountCheckDialog/AccountCheckDialog.tsx]: ooo okay, thanks thats helpful to know

**Review by GeronimoJohn — APPROVED** [review]

**Pull request merged**